src/Game/InventoryList.js

```diff
@@ -0,0 +1,67 @@
+import { supabase } from "../supabase_client";
+import { useEffect, useState } from "react";
+
+const InventoryList = (props) => {
+  const { myPlayerData } = props;
+  const [playerInventory, setplayerInventory] = useState([]);
+  const [newCollectedCardID, setNewCollectedCardID] = useState(0);
+
+  const fetchCardInfo = async () => {
+    const { data } = await supabase
+      .from("CardList")
+      .select("*")
+      .eq("cardID", newCollectedCardID);
+
+    setplayerInventory(() => playerInventory.concat(data));
```

Comment:
If you arent using the previous state value passed to the callback function, then you dont need to pass a function here.

Also just fyi, there no issue with `.concat` at all but I wanna point out another way to achieve the same result is `[...playerInventory, ...data]`. It's good to know the spread operator (but you dont have to change it to this).